### PR TITLE
Bounty Upload | 1-npm-safe-object2 | FRESH - Most PoC Included

### PR DESCRIPTION
**Pull request by benharvie**
If the Sheriffs would kindly validate this vulnerability before we merge it in as a bounty.

Much appreciated,
Ben ❤️

### CODE REVIEW

File: bounties/npm/safe-object2/1/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Overview
+
+`safe-object2` is a Secure operation object `get/set`, this package is vulnerable to Prototype Pollution via. the `setter` function.
+
+# Proof of Concept
+```
+const safeObj2 = require(""safe-object2"");
```
Comment (Mik317) [inline bounties/npm/safe-object2/1/README.md:7]: Replace line with 

```js
const safeObj2 = require("safe-object2");
```

File: bounties/npm/safe-object2/1/vulnerability.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+		"PR": "N",
+		"UI": "N",
+		"S": "U",
+		"C": "H",
+		"I": "H",
+		"A": "H",
+		"E": "",
+		"RL": "",
+		"RC": "",
+		"Score": "9.8"
```
Comment (Mik317) [inline bounties/npm/safe-object2/1/vulnerability.json:35]: Too much `high` as `score`

**Review by Mik317 — APPROVED** [review]
`Valid` :smile: :tada:
I would suggest lowering the `CVSS` score and correcting the PoC (more info in comments 👍 )

Cheers,
Mik

File: bounties/npm/safe-object2/1/README.md (hunk 1 above)
Comment (benharvie) [reply bounties/npm/safe-object2/1/README.md:7]: I will have this amended, cheers! 👍🏼

**Pull request merged**